project/src/test/java/com/google/sps/servlets/QueryServletTest.java

```diff
@@ -16,6 +16,7 @@
 
```

Comment:
I do think there should be a test here that tests that the scorer was actually called and that the results are sorted. WDYT?

Reply:
I think that since this only tests the servlet and not the sorter then it should "trust" the sorter that it does its job once its called with the expected parameters. So I added the assetion that the scorer was called with the right parameters. WDYT?

Reply:
LGTM